jenkins/public_tests_all.Jenkinsfile

```diff
@@ -66,22 +66,28 @@ pipeline {
               unstash "install";
               script {
                 stage("${TEST_SLUG}") {
-                  catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
-                    if ("${TEST_SLUG}" == 'docker build'){
-                      sh "./build_openroad.sh";
-                    } else {
-                      sh 'nice flow/test/test_helper.sh ${TEST_SLUG}';
-                    }
+                  try {
+                      if ("${TEST_SLUG}" == 'docker build'){
+                        sh "./build_openroad.sh";
+                      } else {
+                        sh 'nice flow/test/test_helper.sh ${TEST_SLUG}';
+                      }
+                      currentBuild.result = 'SUCCESS'
+                  } catch (err) {
+                      sh "mkdir -p flow/results/failures"
+                      sh "cp flow/*tar.gz flow/results/failures/."
+                      sh "echo ${TEST_SLUG} | tr ' ' '-' >> flow/results/failures/failed-designs.txt"
```

Comment:
Currently `DESIGN_NAME` and `DESIGN_NICKNAME` overlap and make life more difficult for this PR. Let's drop the `DESIGN_NICKNAME` when it is the same as `DESIGN_NAME`. If we need two variables, let's see if there's a more descriptive name for them.